tests/integration/components/angle-bracket-invocation-test.js

```diff
@@ -350,5 +413,31 @@ module('Integration | Component | angle-bracket-invocation', function(hooks) {
 
       assert.dom('span').hasText('hi martin!');
     });
+
+    test('passing into element - unused with attributes present', async function(assert) {
+      this.owner.register(
+        'template:components/foo-bar',
+        hbs`<span class="hello" ...attributes>hi martin!</span>`
+      );
+
+      await render(hbs`<FooBar />`);
+
+      assert.dom('span').hasText('hi martin!');
+    });
+
+    // This is broken in actual Ember, see
+    // https://github.com/emberjs/ember.js/pull/17533 We're skipping it here too
+    // because otherwise we get test failures on Ember 3.4+, where our polyfill
+    // is not in use.
+    skip('merges attributes in correct priority', async function(assert) {
```

Comment:
Can we tweak to only skip on 3.4+ so we don’t _also_ regress here?

Thinking something like:

```js
if (!hasEmberVersion(3,4)) {
  // test here
}
```

(Can either use ember-compatibility-helpers or @ember/test-helpers/has-ember-version)

Reply:
Sure, I did this.